src/pages/HomePage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import NavBar from '../components/NavBar/NavBar';
+function HomePage() {
+  return (
+    <div>
+      <NavBar />
```

Comment:
NavBar should reside in App.js since it's presumably going to be present in every single page